apps/src/StudioApp.js

```diff
@@ -1616,8 +1612,9 @@ StudioApp.prototype.handleEditCode_ = function (options) {
   }
 
   // droplet may now be in code mode if it couldn't parse the code into
-  // blocks, so update the UI based on the current state:
-  this.onDropletToggle_();
+  // blocks, so update the UI based on the current state (don't autofocus
+  // if we have already created an instructionsDialog at this stage of init)
+  this.onDropletToggle_(!!this.instructionsDialog);
```

Comment:
I prefer the readability of `Boolean` but understand that it actually allocates another object, so is less efficient. I'll probably use `!!` if you all don't mind the lack of readability.